Pod/Classes/Core/RCTScrollView+EnclosingScrollView.m

```diff
@@ -121,32 +122,80 @@ - (RCTDescendantScrollEvent *)scrollEventForScrollView:(UIScrollView *)scrollVie
                                     enclosingScrollViewFrame:enclosingScrollView.frame
                                                     userData:[self valueForKey:@"_userData"]
                                                coalescingKey:coalescingKey];
-    
+
 }
 
 @end
 
+@interface RCTScrollView (RCTEnclosingScrollView)
+- (void)optOutOfParentScrollEvents;
+- (void)optOutOfAllScrollEvents;
+- (void)optInToAllScrollEvents;
+@end
```

Comment:
From a purity perspective, we should probably be swizzling the methods we override and invoke the ‘original’ method as per OOP semantics.

In any case, I'm ok with being pragmatic, but the category name probably doesn’t match what it does well anymore. Perhaps we should just call the category `Artsy` or `Emission`?

Reply:
I'm not sure what you're getting at regarding the overridden methods, but I changed the category to Artsy.